canonical/canonical.com
# WD-12107 Refreshed /lxd page

## Done

- Refreshed /lxd page.
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check if the /lxd page is working fine.
- Check if all the changes are done as according to [copydoc](https://docs.google.com/document/d/1LJ6ajHYAmKdxSNDaD5kbriDod1t7RcmIwVJtevAeKFQ/edit).
- Use this [Figma Design](https://www.figma.com/design/dHmV2b8tplxbBzDW27ypPX/canonical.com%2Flxd?node-id=0-1&t=brs1rvUl7JSpEHe6-0) as visual reference.
- Here is the [demo](https://canonical-com-1284.demos.haus/lxd) generated for the PR.

## Issue / Card

Fixes # [12107](https://warthogs.atlassian.net/browse/WD-12107)
